README.md

```diff
@@ -1,2 +1,48 @@
 # megafuel-go-sdk
-Golang SDK for the eoa-paymaster clients
+
+This is Golang SDK for the MegaFuel clients, it provides a simple way to interact with the MegaFuel.
+
+For more information, please refer to the [API documentation](https://docs.nodereal.io/docs/megafuel-api).
+
+## Quick Start
+1. Install dependency
+ ```shell
+ $ go get -u github.com/nodereal/megafuel-go-sdk
```

Comment:
I believe this is not enough, need add following?
```
replace (
	github.com/cometbft/cometbft => github.com/bnb-chain/greenfield-tendermint v0.0.0-20230417032003-4cda1f296fb2
	github.com/ethereum/go-ethereum => github.com/bnb-chain/bsc v1.4.13
	github.com/syndtr/goleveldb v1.0.1 => github.com/syndtr/goleveldb v1.0.1-0.20210819022825-2ae1ddf74ef7
	github.com/tendermint/tendermint => github.com/bnb-chain/tendermint v0.31.16
)
```

Reply:
no need to replace, just using ethereum isok

Reply:
fixed